app-games/src/main/java/com/aptoide/android/aptoidegames/MainActivity.kt

```diff
@@ -111,6 +111,23 @@ class MainActivity : AppCompatActivity() {
     intent.takeIf { it.isAhab }?.agDeepLink.takeIf { it?.scheme == "promocode" }?.run {
       promoCodeRepository.setPromoCodeApp(PromoCodeApp(host!!, path!!))
     }
+
+    intent.externalUrl?.takeIf { it.scheme in listOf("http", "https") }
+      ?.let { intentWithUrl ->
```

Comment:
The naming is a bit misleading, because this is not an `Intent` anymore but `URI`.
So please rename.